src/util/chaintype.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2023 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <util/chaintype.h>
+
+#include <optional>
+#include <stdexcept>
+#include <string>
+
+std::string ChainTypeToString(ChainType chain)
```

Comment:
I'm not sure on this one. Granted, `std::string ChainName(ChainType chain)` would make it clear for the caller that this is most likely a label associated with the `ChainType`. However, for consistency I would also have to rename the helper functions to `GetChainName()`, and that seems less clear, because it does not have a typed argument allowing the caller to infer the relationship with `ChainType`.